src/super_gradients/examples/deci_loggers_examples/clearml_logger_example.py

```diff
@@ -0,0 +1,31 @@
+from super_gradients.training import Trainer, models
```

Comment:
why is this under deci_logger_example? it is a clearml logger

Reply:
For some reason the folder was not renamed, but I thought about having a single folder "loggers_examples" that would include both deci_logger_example and clearml_logger_example.
What do you think?

Reply:
sounds ok. rename the folder please

Reply:
renamed to `loggers_examples` in 397596cb6620de96b3a88f5f56c33f0d75a52af1